### ISSUE TITLE
Make other tools use apt-fast instead of apt-get

### ISSUE THREAD

**Issue by owais** [ilikenwf/apt-fast#83]
I was wondering if we can make other tools like synaptic or ppa-purge make use of apt-fast instead of apt-get without modifying those programs. Is apt-fast 100% compatible with apt-get? If yes, I think a user could move apt-get binary to apt-get.original and link apt-get -> apt-fast. That would work as  a hack. Any proper way to do this?

**Comment by declension** [comment]
Yes, this is what `update-alternatives` is for. See [this post](http://williamdemeo.github.io/linux/update-alternatives.html) for some examples.
